### PR TITLE
Lens flare effects for strobes, navs, beacons

### PR DESCRIPTION
**Pull request by Uwajimaya**
**Summary of Changes**
- Added lens flare effects to the strobe lights, nav lights, and beacons.
- Changed the tail strobe from a double strobe to a single strobe (real life behavior)
- Made the strobe lights just a hint of blue to make them look more natural

**Screenshots (if necessary)**

**References**
A320 vs 737 light comparison https://www.youtube.com/watch?v=apiD-YMJnoY

**Additional context**
I am creating lighting updates for default aircraft and a few other community mods (TBM mod, C4J, C208). My A320 uses more LED style lights (blue/clear tone), let me know if you'd like that for your A320 as well. https://uwajimaya.github.io/FS2020/

Discord username (if different from GitHub):
Beacon lens flare
Nav   strobe lens flare
Tail lens flare

### CODE REVIEW

**Comment by NathanInnes** [comment]
Do you have a slightly better texture to use for the lens flare? Something less prominent?

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/systems.cfg [hunk 1]
```diff
@@ -15,11 +15,11 @@ lightdef.8 = Type:6#Index:3#LocalPosition:0,0,0#LocalRotation:10,0,55#EffectFile
...
-lightdef.13 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,90#EffectFile:LIGHT_ASOBO_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingLeftOuter#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingLeftOuter
-lightdef.14 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,20#EffectFile:LIGHT_ASOBO_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingLeftInner#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingLeftInner
-lightdef.15 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,-90#EffectFile:LIGHT_ASOBO_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingRightOuter#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingRightOuter
-lightdef.16 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,-20#EffectFile:LIGHT_ASOBO_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingRightInner#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingRightInner
+lightdef.12 = Type:2#Index:0#LocalPosition:-79.15,-0.03,7.77#LocalRotation:0,0,180#EffectFile:LIGHT_A32NX_StrobeTail#Node:LIGHT_ASOBO_StrobeTail#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeTail
```
Comment (lukecologne) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/systems.cfg:18]:
```suggestion
lightdef.12 = Type:2#Index:0#LocalPosition:-79.15,-0.03,7.77#LocalRotation:0,0,180#EffectFile:LIGHT_A32NX_StrobeTail#Node:LIGHT_ASOBO_StrobeTail#PotentiometerIndex:94#EmMesh:LIGHT_ASOBO_StrobeTail
```

**Comment by Uwajimaya** [comment]
Hi! Could you clarify what you mean by "less prominent"? :) Do you mean the halo/circles? Too big or too bright? Or the flashes( the pointy lines)? Too big? To bright? Too many?
I can easily adjust any of them, just let me know what's needed.

**Comment by Uwajimaya** [comment]
Oh, and for the strobes, I recommend looking at it in game, because it goes very quick. And the size gets smaller as you zoom out.

**Comment by UncleClapton** [comment]
I'm not really a fan of this change. The flares don't really fit in with other lights, and feel out of place to me. This is especially true because the tail, landing, wing, and taxi/TO lights didn't get the same treatment. Even if these were included it would still be wildly inconsistent with the rest of the game.

**Comment by lukecologne** [comment]
> 
> 
> I'm not really a fan of this change. The flares don't really fit in with other lights, and feel out of place to me. This is especially true because the tail, landing, wing, and taxi/TO lights didn't get the same treatment. Even if all of those lights were included it would still be wildly inconsistent with the rest of the game.

I agree, with the exception of the beacon lights. IMO, they look fine with the flare

**Comment by Uwajimaya** [comment]
Kein problem, whatever you prefer. I can edit the other lights to have less of a halo, or you can just keep the beacons. Happy to support whatever decision you make.

**Comment by NathanInnes** [comment]
It just looks a lot like FSX

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/systems.cfg [hunk 2]
```diff
@@ -15,11 +15,11 @@ lightdef.8 = Type:6#Index:3#LocalPosition:0,0,0#LocalRotation:10,0,55#EffectFile
...
+lightdef.13 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,90#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingLeftOuter#PotentiometerIndex:94#EmMesh:LIGHT_ASOBO_StrobeWingLeftOuter
+lightdef.14 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,20#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingLeftInner#PotentiometerIndex:94#EmMesh:LIGHT_ASOBO_StrobeWingLeftInner
+lightdef.15 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,-90#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingRightOuter#PotentiometerIndex:94#EmMesh:LIGHT_ASOBO_StrobeWingRightOuter
+lightdef.16 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,-20#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingRightInner#PotentiometerIndex:94#EmMesh:LIGHT_ASOBO_StrobeWingRightInner
```
Comment (lukecologne) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/systems.cfg:22]:
```suggestion
lightdef.13 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,90#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingLeftOuter#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingLeftOuter
lightdef.14 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,20#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingLeftInner#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingLeftInner
lightdef.15 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,-90#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingRightOuter#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingRightOuter
lightdef.16 = Type:2#Index:0#LocalPosition:0,0,6.9#LocalRotation:0,0,-20#EffectFile:LIGHT_A32NX_StrobeDouble#Node:LIGHT_ASOBO_StrobeWingRightInner#PotentiometerIndex:24#EmMesh:LIGHT_ASOBO_StrobeWingRightInner
```

The strobe potentionemter index was reverted back to 24 in #861

**Comment by Uwajimaya** [comment]
> 
> 
> It just looks a lot like FSX

Ouch! :)  But lens flare effects do exist.. https://www.airbus.com/newsroom/news/en/2017/05/red-green-and-white-shedding-light-on-aircraft-illumination.html

**Comment by NathanInnes** [comment]
I could have worded it better with my last comment, if you took offence to it then you shouldn't. What I am trying to say is the texture looks almost like it's from FSX. The lens flare effect you're trying to introduce doesn't fit well with the fundamentals of the game as they look out of place.

**Comment by Uwajimaya** [comment]
No worries, all good. And you are actually not wrong: Asobo is using the same effects system from FSX (with some small frustrating changes). Same file structure, same texture usage, etc. They are reportedly working on something new entirely, but that will probably take quite a while.

**Comment by AdenFlorian** [comment]
can you add before pictures as well please? Also, lens flares are only visible when looking through a lens right? You wouldn't see these would the naked eye.

**Comment by AdenFlorian** [comment]
I also have to agree with @NathanInnes , the flares look out of place, not consistent with the style of the game.

**Comment by Uwajimaya** [comment]
Some bright lights actually can appear as lens flares (your eyes are lenses ;) ) but it's all good. You guys don't feel it's a fit and I totally respect that.  Perhaps once Asobo fixes/replaces the lighting system we'll have an appropriate solution. Thanks and good luck!

**Pull request closed without merging**